independent-publisher-connectors/Pixel Encounter/readme.md

```diff
@@ -0,0 +1,36 @@
+# Pixel Encounter
+
+Simply a web application and an API which generates randomly generated pixel monsters in SVG format. It can be used to create profile pictures or enhance any of your graphic interfaces.
+
+## Publisher: Fördős András
+
+## Prerequisites
+
+Pixel Encounter is an open API and requires no purchase or subscription.
+
+## Obtaining Credentials
+
+There are no credentials needed for this API.
+
+## Supported Operations
+
+### Get a random monster (JSON)
+Get a random pixel monster with its JSON representation.
+
+### Get a monster by ID (JSON)
+Get a monster by ID with its JSON representation.
+
+### List monsters
+Get a paged list with SVG monsters
+
+### Get a random SVG monster (JSON)
+Get an SVG monster with random pattern with JSON representation.
+
+## Known Issues and Limitations
+
+Current version of the connector only supports a subset of the API endpoints. Contact me if you see a need to bring in any of the other ones!
+
+The API itself has some limitations:
+* maximum 2 requests/second tied to the IP address
+* mayimum 10 000 requests/30 days tied to the IP address
```

Comment:
@fordosa90 Thank you so much for submitting a new connector with us. We can't thank you enough.
I wanted to thank you as well for the clean code, it is really fun reviewing your code.
I just have a small comment on the readme file, as we ask for all files to be free of grammatical errors and typos. at line 35 you have "mayimum 10 000 requests/30 days tied to the IP address", is it possible to change it to "maximum 10000 requests/30 days tied to the IP address". 
Thank you so much again.

Reply:
Apologies @Amjed-Ayoub , corrected with a new commit.
Also, included screenshots for verification of functionality.
Thank you for the great and smooth process, it truly is incomparable with how it was at the start of the year, let us keep evolving the platform together!